fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -127,6 +128,14 @@ def self.available_options
                                            UI.user_error!("Please pass a valid platform. Use one of the following: #{available_platforms.join(', ')}") unless available_platforms.map(&:downcase).include?(platform.downcase)
                                          end
                                        end),
+          FastlaneCore::ConfigItem.new(key: :cache_builds,
+                                       env_name: "FL_CARTHAGE_CACHE_BUILDS",
+                                       description: "By default Carthage will rebuild a dependency regardless of whether it's the same resolved version as before. Passing the --cache-builds will cause carthage to avoid rebuilding a dependency if it can. See https://github.com/Carthage/Carthage/blob/master/Documentation/VersionFile.md",
```

Comment:
Let's try to avoid having URLs in the description and add them to the `self.details` instead